psci/PSCi/IO.hs

```diff
@@ -14,8 +14,42 @@
 
 module PSCi.IO where
 
-import System.Directory (createDirectoryIfMissing)
-import System.FilePath (takeDirectory)
+import System.Directory
```

Comment:
I think I'd rather stick to explicit import lists, especially if the code had them to start with. Not only do they help avoid name collisions, they also make it easier to see where a function is coming from.